### ISSUE TITLE
Unable to display only summary in Slack

### ISSUE THREAD

**Issue by apurvanataraj88** [postmanlabs/newman#2028]
Hello @shamasis, @czardoz  

My issue currently is that the entire payload is displayed in Slack along with the requests. Because of the size, it splits into multiple messages, and thus also avoids the `````` symbols to code-ify it. 
![image](https://user-images.githubusercontent.com/45184844/59458617-d2cba000-8de8-11e9-84fd-cd0d0db28b8c.png)

Is there any specific command/parameter I can use to avoid those requests and just get the summary displayed?
I also tried `--reporter-cli-no-failures --reporter-cli-no-assertions --reporter-cli-no-console` mentioned in #291 along with the `newman` command but was not successful.

I am running node 12.4.0 and newman 4.5.0 and running my collection with its associated environment and global variables through the command line.

Regards

**Comment by apurvanataraj88** [comment]
In the beginning, my issue was the same.

1. I first got rid of the validations in the shell script, and hard-coded my `newman` command since that would not be changing for me at all.
2. I was now able to see messages on slack, but it was the entire payload, and not just the summary table. As a temporary workaround, I created another variable `formatted_output` to take in a RegEx and then switched this with `output` in the `curl` command:
My RegEx:
`local formatted_output=$(echo "$output" | grep -A50 '┌─────')`
Curl Command:
`curl -X POST --data-urlencode 'payload={"text": "'"$date"': ```\n'"$formatted_output"'\n```"}' $webhook`
This prints just the summary table to my slack channel.

**Comment by NachoColl** [comment]
@apurvanataraj88 did you find a solution? I'm having the same issue ...

**Comment by apurvanataraj88** [comment]
@NachoColl Yea but it's a work-around

1. I first got rid of the validations in the shell script, and hard-coded my newman command since that would not be changing for me at all.

2. I was now able to see messages on slack, but it was the entire payload, and not just the summary table. As a temporary workaround, I created another variable `formatted_output` to take in a RegEx and then switched this with output in the curl command:
My RegEx:
`local formatted_output=$(echo "$output" | grep -A50 '┌─────')`
Curl Command:
`curl -X POST --data-urlencode 'payload={"text": "'"$date"': ```\n'"$formatted_output"'\n```"}' $webhook`
This prints just the summary table to my slack channel.
